src/matchers/doc.rs

```diff
@@ -106,15 +84,43 @@ fn msooxml(buf: &[u8]) -> Option<DocType> {
     let idx = search(buf, start_offset, 6000);
     match idx {
         Some(idx) => start_offset += idx + 4 + 26,
-        None => return Some(DocType::OOXLM),
+        None => return Some(DocType::OOXML),
     };
 
     let typo = check_msooml(buf, start_offset);
     if typo.is_some() {
         return typo;
     }
 
-    Some(DocType::OOXLM)
+    Some(DocType::OOXML)
+}
+
+#[cfg(feature = "std")]
+fn ole2(buf: &[u8]) -> Option<DocType> {
+    use std::io::Cursor;
+
+    if !compare_bytes(buf, &[0xD0, 0xCF, 0x11, 0xE0, 0xA1, 0xB1, 0x1A, 0xE1], 0) {
+        return None;
+    }
+    if let Ok(file) = cfb::CompoundFile::open(Cursor::new(buf)) {
+        return match file.root_entry().clsid().to_string().as_str() {
```

Comment:
Based on https://stackoverflow.com/questions/29211263/how-to-identify-doc-docx-pdf-xls-and-xlsx-based-on-file-header/48318648#48318648